README.md

```diff
@@ -171,6 +171,16 @@ The below command will build a docker image from source and install it into your
 $ bazel run :bazel-remote-image
 ```
 
+### ARM Support
+
+Bazel remote cache server can be run on an ARM architecture (i.e.: on a Raspberry Pi).
+
+To build for ARM, use:
+
+```bash
+$ docker build --file=Dockerfile_arm64 .
```

Comment:
The "official" way to build bazel-remote is with bazel itself (not with the hard-coded Dockerfile). I wonder if we could make the bazel method work nicely, and document that method instead?

I'm not sure if there's a cleaner way, but we could create bazel-remote-base-arm64 (with `goarch = "arm64"`) and bazel-remote-image-arm64 targets.

Reply:
OK, let me look into this. :+1: